pylint/checkers/len.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2016 Alexander Todorov <atodorov@MrSenko.com>
+
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/master/COPYING
+
+"""Check for incorrect usage of len() in conditions."""
+import itertools
+import astroid
```

Comment:
Not sure I like this being a separate file. We tend to have checkers grouped by a general concept, such as refactoring or checkers related to classes. I would say this belongs into refactoring.py.